belt-block/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## 0.1.1 (2022-09-15)
+- Remove block.rs and move to lib.rs
+- Add test for extended blocks
+- Fix documentation language
```

Comment:
No need to update changelog and crate version, since v0.1.0 will be released only after merge of this PR.

Reply:
Okay, reverted to v 0.1.0